rules/S7119/cfamily/metadata.json

```
@@ -9,16 +9,16 @@
   "tags": [
     "unpredictable"
   ],
-  "defaultSeverity": "Major",
+  "defaultSeverity": "Critical",
   "ruleSpecification": "RSPEC-7119",
   "sqKey": "S7119",
   "scope": "All",
   "defaultQualityProfiles": ["Sonar way"],
   "quickfix": "infeasible",
   "code": {
     "impacts": {
-      "MAINTAINABILITY": "HIGH",
-      "RELIABILITY": "LOW"
+      "MAINTAINABILITY": "LOW",
```

```suggestion
      "MAINTAINABILITY": "MEDIUM",
```
The impact I see is that the code might be working by luck, and some unrelated change breaks it in a non-obvious way, which make changing code complex.